projects/plugins/boost/app/modules/optimizations/page-cache/pre-wordpress/Filesystem_Utils.php

```diff
@@ -4,19 +4,23 @@
 
 class Filesystem_Utils {
 
-	const DELETE_ALL   = 'delete-all'; // delete all files and directories in a given directory, recursively.
-	const DELETE_FILE  = 'delete-single'; // delete a single file or recursively delete a single directory in a given directory.
-	const DELETE_FILES = 'delete-files'; // delete all files in a given directory.
-
+	const DELETE_ALL             = 'delete-all'; // delete all files and directories in a given directory, recursively.
+	const DELETE_FILE            = 'delete-single'; // delete a single file or recursively delete a single directory in a given directory.
+	const DELETE_FILES           = 'delete-files'; // delete all files in a given directory.
+	const REBUILD_ALL            = 'rebuild-all'; // rebuild all files and directories in a given directory, recursively.
+	const REBUILD_FILE           = 'rebuild-single'; // rebuild a single file or recursively rebuild a single directory in a given directory.
+	const REBUILD_FILES          = 'rebuild-files'; // rebuild all files in a given directory.
```

Comment:
`DELETE_ALL` and `DELETE_FILE` appear to do the same thing. The same is true for `REBUILD_ALL` and `REBUILD_FILE`. Any chance we can combine each pair?

Reply:
FILE was supposed to be apply to a single file, but then got applied to a directory, which is what DELETE_ALL does. DELETE_FILE isn't used anywhere either, so these can definitely be simplified.

Reply:
I looked again at the code, and we need to keep DELETE_FILE. It's used when someone leaves a comment, and it's moderated, AND they clicked the "remember me" checkbox. We only want to delete the cache file for them, and nobody else. That [bit of code](https://github.com/Automattic/jetpack/blob/8ffde87d56b4faa0f2c723df78df6fd978c6aa85/projects/plugins/boost/app/modules/optimizations/page-cache/pre-wordpress/Boost_Cache.php#L297) used REBUILD_FILE, but since this file is unique to this user, there's no point rebuilding it. Nobody else will use it, so I replaced it with DELETE_FILE.

While it isn't used now, I'd like to keep REBUILD_FILE too, as it may well be used in the future.